app/models/appointment.rb

```diff
@@ -16,6 +16,8 @@ class Appointment < ApplicationRecord
 
   belongs_to :rebooked_from, class_name: Appointment
 
+  scope :within_48_hours, -> { where('start_at < ?', 2.days.from_now) }
```

Comment:
Perhaps naming this `within_reminder_window` or something that hints at *why* it's 48 hours could be helpful?